<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="ltxt_str"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>输入单个字符</string>
        </property>
        <property name="buddy">
         <cstring>ltxt_char</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>输入字符串</string>
        </property>
        <property name="buddy">
         <cstring>ltxt_str</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_char">
        <property name="text">
         <string>单个字符特性判断</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ltxt_char"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_tr_uni">
        <property name="text">
         <string>每个字符的Unicode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>其他测试和功能</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_tr_latin">
        <property name="text">
         <string>与Latin1 的转换</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_tr_utf8">
        <property name="text">
         <string>与 UTF-16的转换</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_tr_qchar">
        <property name="text">
         <string>QChar比较和替换</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_clear">
        <property name="text">
         <string>清空文本框</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>结果显示</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>QChar特性判断</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="chk_digit">
           <property name="text">
            <string>isDigit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_letter">
           <property name="text">
            <string>isLetter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_letter_num">
           <property name="text">
            <string>isLetterOrNumber</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_upper">
           <property name="text">
            <string>isUpper</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_lower">
           <property name="text">
            <string>isLower</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_mark">
           <property name="text">
            <string>isMark</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_space">
           <property name="text">
            <string>isSpace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_symbol">
           <property name="text">
            <string>isSymbol</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_punct">
           <property name="text">
            <string>isPunct</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
